.github/workflows/pull_request.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         key: ${{ github.job }}-${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
         restore-keys: ${{ github.job }}-${{ runner.os }}-go-build-
     - name: Go vet
-      run: go vet ./...
+      run: GOGC=30 go vet ./...
```

Comment:
LGTM. I think you're right that the failures we've seen are because go vet gets killed for taking up too much memory when it tries to compile the giant OC trees.